### DISCUSSION TITLE
Phone Audio on max after use of AAIdrive

### DISCUSSION THREAD

**Discussion by Cross199104** [BimmerGestalt/AAIdrive#467] [Q&A] [answered]
Hello together.
After I use the AAIdrive App, my Phone audio is on max (not only Bluetooth audio).

Connected over USB and Bluetooth.

Galaxy s10e
AAIdrive 1.3.1
BMW Connected 3.1.0.3071
Spotify 8.6.82.1113

![20211128_170905.jpg](https://user-images.githubusercontent.com/95443362/144592115-338f2650-cefa-4df3-b123-221ea340d1ea.jpg)

Do you know why?
Thank you.

**Comment by hufman** [comment, marked as answer]
Indeed, we do know why this is! Even though AAIdrive has logic to set the audio to max, it only does this after the Bluetooth music connects. BMW Connected (and Classic and MyBMW) all have logic to set the audio to max as soon as the USB connection connects, but without waiting for Bluetooth to connect, which results in exactly what you are experiencing.
Try waiting for Bluetooth to connect before plugging in your phone, which should help in my experience.

**Reply by Cross199104** [reply]
Thanks hufman.
Now it works :-)
